edge-util/test/Microsoft.Azure.Devices.Edge.Util.Test.Common/nunit/EndToEndAttribute.cs

```diff
@@ -0,0 +1,10 @@
+// Copyright (c) Microsoft. All rights reserved.
+namespace Microsoft.Azure.Devices.Edge.Util.Test.Common.NUnit
+{
+    using System;
+    using global::NUnit.Framework;
+
+    [AttributeUsage(AttributeTargets.Class | AttributeTargets.Method, AllowMultiple = true)]
```

Comment:
should we force that all tests in 1 test file should belong to the same category? if so, we can make it only allow to use on Class level, but not method.  And AllowMultiple should be false, right?

Reply:
> should we force that all tests in 1 test file should belong to the same category?
I don't think we need to enforce that.

> AllowMultiple should be false, right?
It's true in all the attributes we currently define. I don't have an opinion one way or the other, so I just followed the existing pattern.

Reply:
Sure. let's unblock this PR.  And we can decide if we need another PR to set "AllowMultiple to false".  I still think it is better to only set in class level to enforce all tests in a test class should belong to the same category.